src/Orleans/Logging/ErrorCodes.cs

```diff
@@ -383,9 +383,9 @@ internal enum ErrorCode
         SiloShutdownEventCreated        = SiloBase + 42,
         SiloShutdownEventOpened         = SiloBase + 43,
         SiloShutdownEventReceived       = SiloBase + 44,
-        SiloLoadedDI                    = SiloBase + 45,
-        SiloFailedToLoadDI              = SiloBase + 46,
-        SiloFileNotFoundLoadingDI       = SiloBase + 47,
+        SiloLoadedDI                    = SiloBase + 45, // Not used anymore
```

Comment:
I did not know what is the policy, since if someone has this code already in the log and later it means something different...I thought they're onetime issued. Like a few lines above there was a comment about an error code which stayed there for backward compat.